spec/helpers/dashboard_helper_spec.rb

```diff
@@ -49,4 +49,60 @@
     end
   end
 
+
+  describe "#number_of_files" do
+    let(:conn) { ActiveFedora::SolrService.instance.conn }
+    let(:user1) { User.new(email:"abc@test") }
```

Comment:
Picking nits here, can you add a space after the colins? https://github.com/bbatsov/ruby-style-guide#spaces-operators